lineman/app/components/group_page/group_page.haml

```diff
@@ -18,10 +18,7 @@
               %group_actions_dropdown{group: 'groupPage.group'}
             .lmo-btn-wrapper
               %notification_volume_dropdown{translate-root: 'group_page', group: 'groupPage.group'}
-          .group-page__nonmember-actions{ng-if: '!groupPage.isMember()'}
-            %a.btn.btn-success.btn-block.lmo-btn-icon{href: '', ng-click: 'groupPage.joinGroup()'}
-              %i.fa.fa-lg.fa-plus{aria-hidden: 'true'}>
-              %span{translate: 'group_page.join_group' }
+          %join_group_button{group: 'groupPage.group'}
```

Comment:
I've been having a good time putting ng-ifs on the directives themselves, so we don't even need to load em up if they're not relevant

```
%join_group_button{group: 'groupPage.group', ng-if: 'canJoinGroup()'}
```